src/components/NewCharacter/NewCharacter.js

```diff
@@ -44,10 +45,20 @@ const NewCharacter = ({ gameId, createCharacter }) => {
   const [characterRace, setCharacterRace] = useState(characterRaceOptions[0].value);
   const [characterOrder, setCharacterOrder] = useState(characterOrderOptions[0].value);
   const [characterMorality, setCharacterMorality] = useState(characterMoralityOptions[0].value);
+  const [characterRoll] = useState(characterProperties.rollCharacter());
```

Comment:
I need the roll to stay stable across updates. If I just do `const characterRoll = characterProperties.rollCharacter();`, anything that causes a re-render will re-do the whole roll and cause the caps and totals to change. Is there a way to do it without `useState` that keeps it stable?

Reply:
Ah ok. There might be another way, but your way is fine. Can we put `setCharacterRoll` in there, or will the linter complain? I've just never seen a `useState` with only half of the tuple.

Reply:
I'll give it a try and see. I was actually thinking adding a card to allow for rerolls if you get a cruddy one, so if add that then it will be there eventually anyway.

Reply:
Yup, the linter complains 😞